libraries/botbuilder-azure/src/cosmosDbStorage.ts

```diff
@@ -261,7 +261,7 @@ function getOrCreateCollection(client: DocumentClient, databaseLink: string, col
             value: string;
         }[];
     } = {
```

Comment:
The SQL Query here "Select r._self" is quite different from the [C# Query](https://github.com/Microsoft/botbuilder-dotnet/blob/34c957e998c03b37585dc993b48148e994134c9c/libraries/Microsoft.Bot.Builder.Azure/CosmosDbStorage.cs#L143). 

This seems like an area where the 2 platforms (and soon 3, with Java coming soon) should be *exactly* aligned. 

`
QueryText = $"SELECT c.id, c.realId, c.document, c._etag FROM c WHERE c.id in ({parameterSequence})"
`

Reply:
This query is used for something different than checking the database existence.
The [equivalent query in JS](https://github.com/Microsoft/botbuilder-js/blob/master/libraries/botbuilder-azure/src/cosmosDbStorage.ts#L115) is exactly the same:
```
query: `SELECT c.id, c.realId, c.document, c._etag FROM c WHERE c.id in (${parameterSequence})`,
```